backend/app/services/jwt_token_service.py

```diff
@@ -0,0 +1,128 @@
+from datetime import datetime, timedelta
+
+from fastapi import Depends, HTTPException, status
+from jose import JWTError, jwt
+from jose.exceptions import ExpiredSignatureError
+from sqlalchemy.orm import Session
+
+from app.core.config import settings
+from app.core.constants import (
+    ACCESS_TOKEN,
+    UNAUTHORIZE_MESSAGE,
+    USER,
+    USER_NOT_FOUND_MESSAGE,
+)
+from app.db.crud import CRUDBase
+from app.db.database import get_db
+from app.interface.jwt_token_interface import JWTTokenInterface
+from app.models.token import Token
+from app.models.user import User
+from app.schema.token_schema import TokenCreate, TokenUpdate
+from app.services.image_service import image_service
+from app.utils.helpers import decode_token
+
+
+class JWTTokenService(JWTTokenInterface, CRUDBase):
+    def __init__(self, db: Session = Depends(get_db)):
```

Comment:
Already discussed, `db` is being instantiated within the class, which can lead to issues with FastAPI's dependency injection.